Apps/WebClient/src/ClientApp/app/components/timeline/laboratory.vue

```diff
@@ -0,0 +1,203 @@
+<style lang="scss" scoped>
+@import "@/assets/scss/_variables.scss";
+
+$radius: 15px;
+
+.timelineCard {
+  border-radius: $radius $radius $radius $radius;
+  border-color: $soft_background;
+  border-style: solid;
+  border-width: 2px;
+}
+
+.entryTitle {
+  background-color: $soft_background;
+  color: $primary;
+  padding: 13px 15px;
+  font-weight: bold;
+  margin-right: -1px;
+  border-radius: 0px $radius 0px 0px;
+}
+
+.icon {
+  background-color: $primary;
+  color: white;
+  text-align: center;
+  padding: 10px 0;
+  border-radius: $radius 0px 0px 0px;
+}
+
+.leftPane {
+  width: 60px;
+  max-width: 60px;
+}
+
+.detailsButton {
+  padding: 0px;
+}
+
+.detailSection {
+  margin-top: 15px;
+}
+
+.commentButton {
+  border-radius: $radius;
+}
+
+.newComment {
+  border-radius: $radius;
+}
+
+.collapsed > .when-opened,
+:not(.collapsed) > .when-closed {
+  display: none;
+}
+</style>
+
+<template>
+  <b-col class="timelineCard">
+    <b-row class="entryHeading">
+      <b-col class="icon leftPane">
+        <font-awesome-icon :icon="entryIcon" size="2x"></font-awesome-icon>
+      </b-col>
+      <b-col class="entryTitle">
+        {{ entry.testType }}
+      </b-col>
+    </b-row>
+    <b-row class="my-2">
+      <b-col class="leftPane"></b-col>
+      <b-col>
+        <b-row>
+          <b-col>
+            {{ entry.loincName }}
+          </b-col>
+        </b-row>
+        <b-row>
+          <b-col>
+            <div class="d-flex flex-row-reverse">
+              <b-btn
+                v-b-toggle="'entryDetails-' + index + '-' + datekey"
+                variant="link"
+                class="detailsButton"
+                @click="toggleDetails()"
+              >
+                <span class="when-opened">
+                  <font-awesome-icon
+                    icon="chevron-down"
```

Comment:
We want chevron-up when opened and chevron-down when closed

Reply:
I think that is how medication works now. Thats where I got it from